specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2020-11-01-preview/containerregistry.json

```diff
@@ -3944,6 +3944,10 @@
             "name": "PublicNetworkAccess",
             "modelAsString": true
           }
+        },
+        "zoneRedundancy": {
```

Comment:
Please update the registry examples for `2020-11-01-preview` to show this property in the response.